CHANGES.md

```diff
@@ -77,6 +77,7 @@ Parser:
 
 Grammars:
 
+- fix(ps1) Flag highlighted incorrectly (#3167) [Josh Goebel][]
```

Comment:
Please move this up to the top under `## Version 11.1.0 ` and shouldn't it have your name, not mine?

Reply:
Ping.